src/normalize/entities.ts

```diff
@@ -410,7 +422,8 @@ export const Filter = new Entity({
 export const InviteReply = new Entity({
 	compNum: 'componentNum',
 	m: ['message', MessageInfo],
-	rt: 'replyType'
+	rt: 'replyType',
+	exceptId: ['specificInvite', InstanceDate]
```

Comment:
This is named `exceptId` on another `Invite` entity in the file, can we change this to be `exceptId` to be consistent?

Reply:
Resolved above review comment.